modules/server/src/network/aggregations/index.ts

```diff
@@ -1,22 +1,39 @@
 import { SupportedAggregation, SUPPORTED_AGGREGATIONS } from '../common';
 import { Aggregations, NetworkAggregation, NumericAggregations } from '../types';
 
-// TODO: implement
-export const resolveAggregations = (networkResults: void) => null;
+/**
+ * Pick each field from network result aggregations and reduce into single aggregation
+ *
+ * @param networkResults
+ * @param rootQueryFields
+ */
+export const resolveAggregations = (networkResults, rootQueryFields) => {
+	const resolvedNetworkAggregations = rootQueryFields.map((fieldName) => {
+		const fieldAggregations = networkResults.map((networkResult) => {
+			const documentName = Object.keys(networkResult)[0];
+			return networkResult[documentName][fieldName];
+		});
+		const aggregationType = fieldAggregations[0].__typename;
+		const resolvedAggregation = resolveToNetworkAggregation(aggregationType, fieldAggregations);
+		return { fieldName: fieldName, aggregation: resolvedAggregation };
+	});
+
+	return resolvedNetworkAggregations;
+};
 
 /**
  * Resolve aggregation based on aggregation type
  * @param type
  * @param aggregations
  */
-export const resolveToNetworkAggregation = <T>(
+export const resolveToNetworkAggregation = (
 	type: SupportedAggregation,
 	aggregations: Aggregations[],
 ) => {
```

Comment:
Wouldn't mind declaring a response type here if you know what should be returned, since we have multiple return statements this would validate that they are returning consistent types. Also helps me as I'm reading to know what the output of this function will be.